index.bs

```diff
@@ -1491,6 +1500,10 @@ To <dfn for="WebTransportSendStream">write</dfn> |chunk| to a {{WebTransportSend
      [=WritableStream/Error | errored=] nor blocked by [=flow control=], have been
      sent.
 
+    Note: We access |stream|.{{[[SendOrder]]}} [=in parallel=] here. User agents SHOULD
+    respond to live updates of these values during sending, though the details are
+    [=implementation-defined=].
```

Comment:
So the implementation of this will be asynchronous in practice, but the interface is simple.  That means that browsers will need to retain a value for the stream objects it maintains.  It then needs to propagate changes asynchronously to the stack, as this mentions.

The alternative is to describe two values, with the one that is used here being updated asynchronously (and potentially fallibly, with no error reporting).